submitit/helpers.py

```diff
@@ -220,8 +220,31 @@ def __exit__(self, *args):
         os.chdir(self.original_dir)
 
 
+def _default_monitor_callable(
+    monitoring_start_time: float, n_jobs: int, state_jobs: tp.Dict[str, tp.Set[int]]
+):
+    run_time = time.time() - monitoring_start_time
+    date_time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    failed_job_indices = sorted(state_jobs["FAILED"])
+    n_chars = len(str(n_jobs))
+
+    print(
+        f"[{date_time}] Launched {int(run_time / 60)} minutes ago,",
+        f"{len(state_jobs['RUNNING']):{n_chars}}/{n_jobs} jobs running,",
+        f"{len(failed_job_indices):{n_chars}}/{n_jobs} jobs failed,",
+        f"{len(state_jobs['DONE']) - len(failed_job_indices):{n_chars}}/{n_jobs} jobs done",
+        flush=True,
+    )
+
+    if len(failed_job_indices) > 0:
+        print(f"[{date_time}] Failed jobs, indices {failed_job_indices}", flush=True)
+
+
 def monitor_jobs(
-    jobs: tp.Sequence[core.Job[core.R]], sleep_time_s: float = 30, test_mode: bool = False
+    jobs: tp.Sequence[core.Job[core.R]],
+    sleep_time_s: float = 30,
+    test_mode: bool = False,
+    monitor_callable: tp.Callable = _default_monitor_callable,
```

Comment:
could we rename `monitor_callable` into something less abstract ? maybe `custom_logging` ?